R/tm_data_table.R

```diff
@@ -305,36 +323,65 @@ srv_data_table <- function(id,
                            if_distinct,
                            dt_args,
                            dt_options,
-                           server_rendering) {
+                           server_rendering,
+                           decorators) {
   moduleServer(id, function(input, output, session) {
     iv <- shinyvalidate::InputValidator$new()
     iv$add_rule("variables", shinyvalidate::sv_required("Please select valid variable names"))
     iv$add_rule("variables", shinyvalidate::sv_in_set(
       set = names(isolate(data())[[dataname]]), message_fmt = "Not all selected variables exist in the data"
     ))
     iv$enable()
+    # teal::validate_inputs(iv)
```

Comment:
This won't pass `linter`, can you remove it?

Reply:
I think I need to uncomment, this is a remaining from testing phase

Reply:
@averissimo when I uncomment this line the module does not produce an output : D

Reply:
Solved in f94a6586